commercial/app/model/capi/Lookup.scala

```diff
@@ -26,11 +26,35 @@ class Lookup(contentApiClient: ContentApiClient) extends Logging with implicits.
     }
   }
 
+  def shortUrlToShortId(shortUrl: String): String = {
+    /*
+        Date: 02nd Dec 2020
+        id: 288767d7-ba82-4d67-8fb3-9139e67b0f2e
+
+        CAPI (recently) announced that we would be moving
+
+        from
+          https://gu.com/p/abc
+
+        to
+          https://www.theguardian.com/p/abc
+
+        for "short" urls.
+
+        Introducing this to handle gracefully both the old and new convention without having to
+        worry when the change will actually happen. This function can be simplified in the future when the migration
+        has completed.
+     */
+    shortUrl
+      .replaceFirst("^[a-zA-Z]+://gu.com/", "")
+      .replaceFirst("^[a-zA-Z]+://www.theguardian.com/", "")
+  }
+
   def contentByShortUrls(
       shortUrls: Seq[String],
   )(implicit executionContext: ExecutionContext): Future[Seq[ContentType]] = {
     if (shortUrls.nonEmpty) {
-      val shortIds = shortUrls map (_.replaceFirst("^[a-zA-Z]+://gu.com/", "")) mkString ","
+      val shortIds = shortUrls.map(shortUrlToShortId) mkString ","
```

Comment:
If we’re going for dots, let’s be consistent in this line.

```suggestion
      val shortIds = shortUrls.map(shortUrlToShortId).mkString(",")
```